source/contributing/style-guide.md

```diff
@@ -16,119 +15,104 @@ The same holds for communication directed to contributors and maintainers:
+[Wikipedia URLs to specific page versions]: https://en.wikipedia.org/wiki/Wikipedia:Linking_to_Wikipedia#Permanent_links_to_old_versions_of_pages
+[Internet Archive "Save Page Now" for persisting web pages]: https://web.archive.org/save
+
+## Markdown
 
-```
-## This is a section
-Lorem ipsum dolor sit amet, [consectetur][adipiscing] elit. Suspendisse rutrum ligula porta, condimentum dui dignissim, imperdiet mi. Sed interdum lacus nec varius posuere. Duis auctor varius purus, ut ornare purus tempus eu. Aliquam erat volutpat. Etiam eget nunc malesuada, elementum neque eget, mollis metus. Nulla suscipit felis nec accumsan fermentum.
+- Write one sentence per line.
 
-Integer volutpat erat sem, non varius turpis facilisis eu. Nam eu [ullamcorper][magna]. Morbi iaculis vel urna in condimentum. Lorem ipsum dolor sit amet, consectetur adipiscing elit. Phasellus nec tellus sit amet tellus venenatis porta in et ex. Nunc sodales nisl magna, at dictum diam sollicitudin id. Orci varius natoque penatibus et magnis dis parturient montes, nascetur ridiculus mus. Donec facilisis, sapien eu faucibus iaculis, nibh nibh condimentum enim, nec egestas turpis erat nec libero. Curabitur ut tincidunt odio. Praesent sed tincidunt tortor.
+  This makes long sentences immediately visible, and makes it easier to review changes and make suggestions.
 
-[adipiscing]: example.com
-[magna]: example.com
-```
+  The rule is unambiguous and does not require tooling support to be applied easily.
+  [Here is a discussion of different line wrapping styles.]
+
+  [Here is a discussion of different line wrapping styles.]: https://web.archive.org/web/20220519121408/https://mtsknn.fi/blog/4-1-wrapping-styles-for-markdown-prose-and-code-comments/
+
+- Use [reference links](https://github.github.com/gfm/#reference-link) where needed, either to keep the plain text containing many references readable or to reuse the same link multiple times.
+
+  Keep reference link definitions close to their use site so they are easy to find without additional tooling.
+
+# Licensing and attribution
+
+When opening pull requests with your own contributions, you agree to licensing your work under [CC-BY-SA 4.0](https://creativecommons.org/licenses/by-sa/4.0/).
+
+When adding material by third parties, make sure it has a license that permits this.
+In that case, unambiguously state source, authors, and license in the newly added material.
+Notify the authors *before* using their work.
+
+[Add the original author as co-author](https://docs.github.com/en/pull-requests/committing-changes-to-your-project/creating-and-editing-commits/creating-a-commit-with-multiple-authors) to the first commit of your pull request, which should contain the original document verbatim, so we can track authorship and changes through version history.
+
+Using free licenses other than CC-BY-SA 4.0 is possible for individual documents, and by contributing changes to those documents you agree to license your work accordingly.
```

Comment:
I don't understand what this is saying. Are we telling users that they can submit stuff for merging that's under a different license than CC-BY-SA 4.0? If so, why?

Reply:
Yes.

```suggestion
Adding foreign documents that have free license other CC-BY-SA 4.0 is possible.
Add a license note to the respective document in that case.
By contributing changes to that document you agree to follow the terms of that license.

We would prefer re-licensing to CC-BY-SA 4.0 to keep it consistent and easier to work with, but this requires active cooperation from authors and is therefore not always possible or practical.
```

Did not follow our own style guides there... Hope the change answers the question.